gulpfile.js

```diff
@@ -75,7 +75,10 @@ gulp.task('depcheck', () =>
       // "@types/*" dependencies are type declarations that are automatically
       // loaded by TypeScript during build. depcheck can't detect this
       // so we ignore them here.
-      ignoreMatches: ['@types/*']
+      // "hydrolosis" is being incorrectly matched because it is no longer
+      // used directly in code. TODO: remove from this list once we remove
```

Comment:
only type definitions for deprecations in `analyzer.ts`, which I think is why it can't find it. Looks like a bug in depcheck, I'll try to dig in a little when I get some time